OAuth docs tweaks (#2679)
bluesky-social/atproto: packages/api/OAUTH.md

# OAuth Client Quickstart

This document describes how to implement OAuth based authentication in a
browser-based Single Page App (SPA), to communicate with
[atproto](https://atproto.com) API services.

## Prerequisites

> - You need a web server, or at very least a static file server, to host your SPA.

Reviewer: ```suggestion
- You need a web server - or at the very least a static file server - to host your SPA.
```

> [!TIP]
>
> You can use a service like [GitHub Pages](https://pages.github.com/) to host
> your client metadata and SPA for free.

- You must be able to build and deploy a SPA to your server.

## Step 1: Create your client metadata

Based on your hosting server endpoint, you will first need to choose a
`client_id`. That `client_id` will be used to identify your client to

> Authorization Servers. A `client_id` must be a URL that points to a JSON file
> that contains your client metadata. The client metadata **must** contain a
> `client_id` that is the URL used to access the metadata.

Reviewer: ```suggestion
Authorization Servers. A `client_id` must be a URL pointing to a JSON file
which contains your client metadata. The client metadata **must** contain a
`client_id` that is the URL used to access the metadata.
```

> [!NOTE]
>
> To mitigate phishing attacks, the Authentication Server will typically _not_
> display the `client_uri` or `logo_uri` to the user. If you don't see your logo

> > or client name during the authentication process, don't worry, this is normal.

Reviewer: ```suggestion
> or client name during the authentication process, don't worry. This is normal.
```

> The `client_name` _is_ generally displayed for all clients.

Upload this JSON file so that it is accessible at the URL you chose for your
`client_id`.

> ## Step 2: Setup you SPA 

Reviewer: ```suggestion
## Step 2: Setup your SPA 
```

> [!CAUTION]
>
> Using Bluesky-hosted services for handle resolution (eg, the `bsky.social`
> endpoint) will leak both user IP addresses and handle identifier to Bluesky,
> a third party. While Bluesky has a declared privacy policy, both developers
> and users of applications need to be informed of and aware of the privacy
> implications of this arrangement. Application developers are encouraged to
> improve user privacy by operating their own handle resolution service when
> possible. If you are a PDS self-hoster, you can use your PDS's URL for
> `handleResolver`.

The `oauthClient` is now configured to communicate with the user's

> Authorization Service. We can now initialize it in order to detect if the user

Reviewer: ```suggestion
Authorization Service. You can now initialize it in order to detect if the user
```
